README.md

```diff
@@ -21,7 +23,51 @@ sbt antlr4:antlr4Generate # Generates source code for IntellIJ IDEA
 sbt clean compile test
 ```
 
-See the wiki for detailed documentation.
+## Running Locally
+
+### Requirements:
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) (4GB+, 8GB recommended)
+* Broad internal internet connection (or VPN, non-split recommended)
+* Render the local configuration files. From the root of the [firecloud-develop](https://github.com/broadinstitute/firecloud-develop) repo, run:
+```sh
+./firecloud-setup.sh
```

Comment:
I tried these step to resolve the database host name not set errors from above, but I don't see such a file in the root of the firecloud-develop repo.

Reply:
I've fixed the path for this file in the latest commit